### DISCUSSION TITLE
Addition of external staging sites to prevent unnecessary license slots being used.

### DISCUSSION THREAD

**Discussion by benjaminfphillips** [elementor/elementor#21693] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I've been an Elementor customer for a couple of years, thankfully I am on the GF'd 1000 license plan, however its come to light recently that due to my workflow, I'm using 2 licenses per site.

Before DNS control my host (20i) one of the most popular reseller account providers in the UK use stackstaging.com for staging sites. Once I have DNS control I activate the license on domain.com and the stackstaging becomes staging.domain.com. The stackstaging site is never used again but still consumes a license slot. Obviously I can deactivate this but I am concerned people will get caught within your license hopping crackdown in cases similar to mine.

### Describe the solution you'd like

We could add external staging providers to your site's approved list. At the minute, the list is rather small.

In my example, stackstaging.com is clearly for staging. The clue is within the domain itself, the sites end up being site-co-uk.stackstaging.com for example.

I'm sure stackpath provide staging as well as CDN, etc to many other hosting providers as well and it seems really unfair to use an extra license, for what is obviously a staging site.

### Describe alternatives you've considered

N/A

**Comment by VerdiH** [comment]
@nicholaszein maybe you can help?
